SpecFlowSolar/StepDefinitions/BaseClass.cs

```diff
@@ -1,26 +1,21 @@
 ﻿using OpenQA.Selenium;
-using OpenQA.Selenium.Chrome;
+using SpecFlowSolar.Pages;
 
 namespace SpecFlowSolar.StepDefinitions
 {
     public abstract class BaseClass
     {
         public IWebDriver _driver;
-        public IJavaScriptExecutor _js;
 
-        [BeforeScenario]
-        public void Setup()
-        {
-            var options = new ChromeOptions();
-            options.AddArgument("window-size=1400,1200");
-            _driver = new ChromeDriver(options);
-            _js = (IJavaScriptExecutor)_driver;
-        }
+        public CartPage cartPage;
```

Comment:
this is incorrect to define variables here and assign values in particular Steps.
Someday you will have null reference exception because in some steps there will be no page = new Page() and you will call this page while it is null.
Either define here and make all 
            homePage = new HomePage(_driver);
            cartPage = new CartPage(_driver);
            catalogPage = new CatalogPage(_driver);
here.
Or define separate 
private CartPage cartPage;
cartPage = new CartPage(_driver);
etc.
in each Steps file